X/XDiag/build_tarballs.jl

```diff
@@ -42,7 +42,9 @@ if [[ "${target}" == x86_64-apple-* ]]; then
     export MACOSX_DEPLOYMENT_TARGET=10.14
 fi
 
-cmake -S . \
+
+if [[ "${target}" == *-apple-* ]]; then
```

Comment:
Instead of duplicating build configurations, please do something like
```sh
if [[ "${target}" == *-apple-* ]]; then
    FLAGS=(...)
fi
cmake ... \
    ... \
    "${FLAGS[@]}"
```

Reply:
I added this now in the new PR. Thanks for the suggestion